index.js

```diff
@@ -55,6 +56,7 @@ module.exports = {
       }
 
       var linted = trees.map(function(tree) {
+        let filteredTreeToBeLinted = new Funnel(tree, { exclude: '**/*.js' });
         return new ScssLinter(mergeTrees([tree]), this.scssLintOptions);
```

Comment:
Was this perhaps meant to be `return new ScssLinter(mergeTrees([filteredTreeToBeLinted]), this.scssLintOptions);`?